flutter_todo/lib/components/todo_item.dart

```diff
@@ -0,0 +1,109 @@
+import 'package:flutter/material.dart';
+import 'package:flutter_todo/components/modify_item.dart';
+import 'package:flutter_todo/components/widgets.dart';
+import 'package:provider/provider.dart';
+import 'package:flutter_todo/realm/schemas.dart';
+import 'package:flutter_todo/realm/realm_services.dart';
+
+enum MenuOption { edit, delete }
+
+class TodoItem extends StatelessWidget {
+  final Item item;
+
+  const TodoItem(this.item, {Key? key}) : super(key: key);
+
+  @override
+  Widget build(BuildContext context) {
+    final realmServices = Provider.of<RealmServices>(context);
+    bool isMine = (item.ownerId == realmServices.currentUser?.id);
+    return item.isValid
+        ? ListTile(
+            leading: Checkbox(
+              checkColor: Colors.white,
+              fillColor: MaterialStateProperty.resolveWith(getColor),
+              value: item.isComplete,
+              onChanged: (bool? value) async {
+                if (item.ownerId == realmServices.currentUser?.id) {
+                  await realmServices.updateItem(item, isComplete: value ?? false);
+                } else {
+                  showError(context, "Edit not allowed!", '''You are not allowed to edit tasks
+            that don't belog to you.''');
```

Comment:
nit: why `'''` and line wrap instead of just putting the whole string on a new line?

Reply:
It looks better in the UI, because it adds a new line.

Reply:
Fixed in https://github.com/mongodb-university/realm-template-apps/pull/110/commits/bde8d827ad01dbd8ff0ebd45a682a4048410c2b9